scripts/publish.sh

```diff
@@ -123,6 +123,34 @@ check_parameter() {
 	pull_ecr $repo_uri $region
 }
 
+sync_latest_image() {
+	region=${1}
+	account_id=${2}
+	sha1=$(docker pull amazon/aws-for-fluent-bit:latest | grep sha256: | cut -f 3 -d :)
+
+	endpoint='amazonaws.com'
+	if [ "${1}" = "cn-north-1" ] || [ "${1}" = "cn-northwest-1" ]; then
+		endpoint=${endpoint}.cn
+	fi
+
+	repoList=$(aws ecr describe-repositories --region ${region})
+	repoName=$(echo $repoList | jq .repositories[0].repositoryName)
+	if [ "$repoName" = '"aws-for-fluent-bit"' ]; then
+		pull_ecr ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit:latest ${region}
+		sha2=$(docker inspect --format='{{index .RepoDigests 0}}' ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit:latest)
+	else
+		sha2='repo_not_found'
+	fi
+
+	docker images
+	match_two_sha $sha1 $sha2
+
+	if [ "$IMAGE_SHA_MATCHED" = "FALSE" ]; then
+		publish_ecr ${region} ${account_id}
+		publish_ssm ${region} ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit
+	fi
```

Comment:
As we discussed yesterday, there's a slight problem with this.

Consider the following scenario:
1. A new image is published by the main pipeline
2. The sync job runs and syncs the new image
3. But line 150, the publish_ssm step fails for some reason, say a random network failure.
4. Next time the sync job runs, it will check if the image sha's match. They will, so it will not publish. SSM will never be updated.